src/libraries/listener.library.ts

```diff
@@ -15,26 +33,80 @@ export interface IKeyboardShortcutListenerConstructorObject
 }
 
 export interface IKeyboardShortcutListener {
```

Comment:
we should have an @description block on this object with something like "represents a shortcut listener object"